src/styles/catalog_cards_styles.scss

```diff
@@ -0,0 +1,33 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Manrope, sans-serif;
+  font-family: Roboto, sans-serif;
```

Comment:
There are two font-family properties declared one after the other in the body selector. The second one will overwrite the first one. If you want to use them as fallbacks, list them in one font-family property.